CardParts/src/Classes/Card Parts/CardPartAttributedTextView.swift

```diff
@@ -206,30 +206,20 @@ public class CardPartAttributedTextView: UIView, CardPartView {
     func setDefaultsForType(_ type: CardPartAttributedTextType) {
         switch type {
         case .small:
-            DispatchQueue.main.async { [weak self] in
-                self?.font = CardParts.theme.smallAttributedTextFont
-                self?.textColor = CardParts.theme.smallAttributedTextColor
-            }
+            font = CardParts.theme.smallAttributedTextFont
+            textColor = CardParts.theme.smallAttributedTextColor
```

Comment:
@jl-codes why is it removed ?

Reply:
In the current form, we are unable to override the font and textColor attributes. Removing the DispatchQueue from this allows for us to override these settings in the view controller.

Reply:
In the current form, we are unable to override the font and textColor attributes. Removing the DispatchQueue from this allows for us to override these settings in the view controller.

Reply:
In didSet of font and didSet of textColor we are already Dispatching on main thread. The problem is that it puts it on the main thread twice